main/acle.rst

```diff
@@ -1794,6 +1796,9 @@ Identification extension is used to protect branch destinations by default.
 The protection applied to any particular function may be overriden by
 mechanisms such as function attributes.
 
+``__ARM_FEATURE_BTI`` is defined to 1 if Branch Target Identification
```

Comment:
It is not clear from reading the section on Branch Target Identification why we need two separate macros. Maybe you could add a design document explaining why?